mta/mta.sw.yaml

```diff
@@ -154,12 +155,16 @@ states:
           refName: createNotification
           arguments:
             title: "\"MTA analysis report for \" + .application.repository.url"
-            message: '"<a href=\"" + $SECRET.mta_url+ "/hub/applications/" + (.application.id|tostring)+ "/bucket/" + .data.output + "\">MTA Analysis Rerport</a>"'
+            message: '"<a href=\"" +  + "\">MTA Analysis Rerport</a>"'
```

Comment:
```
Perhaps the title can be more generic such as:
```
title: \"MTA analysis completed\"
message: "\"MTA analysis is completed and the report for \" + .application.repository.url + \" is ready \""
```